.package.json.last_installed

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
Please remove this file from Git. It shouldn't be committed. Use `git rm --cached .package.json.last_installed` to remove it only from Git. [More information about `git rm` here](https://www.git-tower.com/learn/git/commands/git-rm/)